<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>omni::ui::Scene</class>
 <widget class="QWidget" name="omni::ui::Scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="horizontalSpacing">
    <number>-1</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="2" column="1">
    <widget class="QComboBox" name="boxFrustumViewMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Inside</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Outside</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Both sides</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Projector View</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Display</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frameDisplay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkInput">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkMeasures">
        <property name="text">
         <string>Measures</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkProjectors">
        <property name="text">
         <string>Projectors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkGrid">
        <property name="text">
         <string>Grid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Scene Size</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="boxSceneSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
